### DISCUSSION TITLE
problem with routing from static to dynamic id page

### DISCUSSION THREAD

**Discussion by megvargian** [nuxt/nuxt#21951] [Questions] [answered]
we’re having an issue with routing from a specific-id page to a dynamic one:
Structure:
page/[id]/index.vue
pages/123/example.vue
Problem is, we can use router.push to go from pages/[id]/index.vue to pages/123/example.vue, but we cannot go from pages/123/example.vue to pages/[id]/index.vue .
Another problem, if we route to pages/123/example.vueand reload the page, it throws an error 404.
What’s the problem? and how can we solve it?

**Comment by warflash** [comment, marked as answer]
Could you add a stackblitz of the behaviour you're seeing? The description is kind of difficult to follow

**Reply by megvargian** [reply]
it is something like this https://stackblitz.com/edit/stackblitz-starters-rv4y7o?file=pages%2F1234%2Findex.vue  if you do have more question let me know @warflash (the '1234' folder is a specific clientId)

**Reply by warflash** [reply]
Thanks, sorry if I wasn't clear. I was more refering to the official nuxt reproduction template https://stackblitz.com/github/nuxt/starter/tree/v3/ so we can run the nuxt app and see what the issue might be

**Comment by megvargian** [comment]
Sorry, I don't have permission to share with you the files of the project. thank you for the help. @warflash

**Reply by TheAlexLichter** [reply]
@megvargian Then try to reproduce it without including "business/company details & logic" ☺️ 
Otherwise it is not really possible to follow-up.
